src/atn-editor/store/reducers/featureReducer.ts

```diff
@@ -0,0 +1,28 @@
+import { AnyAction } from 'redux';
+
+export interface StateType {
```

Comment:
Same here. Can we use proper names for all the state types?
```suggestion
export interface FeatureReducerState {
```